framework/util/fix_test_suite.pl

```diff
@@ -228,12 +228,17 @@ =head2 Test Suites
 
 =head2 Logging
 
-This script logs all information to fix_tests.log in the test suite directory
-F<SUITE_DIR>.
+This script logs all compilation and execution information to
+comp_tests.log and run_tests.log respectively in the test suite
+directory F<SUITE_DIR>. It also logs all produced information to
```

Comment:
I like the separation. What about the following:
* fix_test_suite.log only provides summary information (such as how many tests were removed and why) and references the other two log files where appropriate.
* comp_tests.log provides detailed information about the test-compilation steps.
* run_tests.log provides detailed information about the test-execution steps.

Regarding the naming of the log files, what about:
* fix_test_suite.summary.log
* fix_test_suite.compile.log
* fix_test_suite.run.log

Feel free to suggest other naming schemes.